accounts/management/commands/update_adopters.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand, CommandError
+from accounts.salesforce import Salesforce
+from pages.models import Organizations
+
+
+class Command(BaseCommand):
+    help = "update adopters from salesforce.com"
+
+    def add_arguments(self, parser):
+        parser.add_argument('page_id', type=int)
```

Comment:
Maybe instead of making this an argument - you can query pages for the adopter page and use the ID - as an intermediate solution?
Something like:
```
from pages.models import Adopters
Adopters.objects.all()[0]
````